ui_framework/doc_site/src/views/gallery/gallery.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import {
+  KuiGallery,
+  KuiGalleryButton,
+  KuiGalleryButtonIcon,
+  KuiGalleryButtonImage,
+  KuiGalleryButtonLabel
+} from '../../../../components';
+
+export default () => {
+  /**
+  * These styles are just for demonstration purposes. It is recommended to use
+  * properly named classes instead of CSS properties in production code.
+  */
+  const imageStyle = {
+    backgroundColor: 'lightgray'
+  };
+
+  return (
+    <div>
+      <div className="kuiVerticalRhythm">
+        <div className="kuiSubTitle">
+          Some items
+        </div>
+
+        <KuiGallery>
+          <KuiGalleryButton href="#">
+            <KuiGalleryButtonImage style={imageStyle}/>
+
+            <KuiGalleryButtonLabel>
+              Item A
+            </KuiGalleryButtonLabel>
+
+            <KuiGalleryButtonIcon className="fa-flask"/>
```

Comment:
Can we add `aria-hidden="true"` here too? We're starting to improve Kibana's accessibility, and this will document the best practice of hiding uninformative text elements from screen readers.